### DISCUSSION TITLE
关于IC2中尝试引爆<工业TNT>而导致的游戏崩溃问题

### DISCUSSION THREAD

**Discussion by WaterDouble** [IndustrialCraftMC/IndustrialCraft2#22] [【漏洞反馈】Bugs Report]
[错误报告-2023-5-6_22.22.59.zip](https://github.com/AbbottMc/IndustrialCraft2/files/11412620/-2023-5-6_22.22.59.zip)

用的1.19.2 forge-43.2.0 Op-I1-pre4

描述：点然<工业TNT>后游戏现实无响应，而后直接崩溃，重新进入世界后可以正常看见其爆炸所带来的伤害及掉落物反馈，由于游戏崩溃，故此无爆炸音效反馈。

**Comment by AbbottMc** [comment]
该内容与已有的Bug反馈重复，详情见：https://github.com/AbbottMc/IndustrialCraft2/discussions/8#discussion-5159733
